### ISSUE TITLE
svnX fails on Diff (FileMerge) when used in a multi-user environment (file permissions)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#102]
```
What steps will reproduce the problem?
1. Use svnX on a multi-user computer.
2. Use Diff with one user.
3. Use Diff with another user.

What is the expected output?

Diff/FileMerge works properly for all users

What do you see instead?

Permissions error for the second user.  Owner of /tmp/svnx is the first user, 
second user does not have permission to edit it.

What version of the product are you using? On what operating system?

1.2, 10.6.3.

Please provide any additional information below.


```

Original issue reported on code.google.com by `andrew%c...@gtempaccount.com` on 21 Jun 2010 at 4:16

**Comment by GoogleCodeExporter** [comment]
```
Please try this patch (on svnX.app/Contents/Resources/svndiff.sh):

===================================================================
--- Resources/svndiff.sh    (revision 147)
+++ Resources/svndiff.sh    (working copy)
@@ -21,6 +21,7 @@
 while ([ ! -d $dir ] && ! mkdir $dir &> /dev/null); do
    dir="/tmp/svnx$((n++))"
 done
+chmod -f a+rw $dir

 # Sometimes, the temp file created by svn diff is deleted before
 # opendiff had a chance to open it...
===================================================================

```

Original comment by `chris...@gmail.com` on 21 Jun 2010 at 5:41
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
That seems to have worked.
```

Original comment by `andrew%c...@gtempaccount.com` on 21 Jun 2010 at 5:53

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3b1.
Similarly for the Blame command.
```

Original comment by `chris...@gmail.com` on 16 Sep 2010 at 12:29
- Changed state: **Fixed**
